### PR TITLE
Backup docs

### PR DESCRIPTION
**Pull request by lllama**
Detail backing up Elastic using built in functionality.

### CODE REVIEW

File: docs/backups.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Backing up LME Logs
+
+Logs are backed up using the built-in Elastic facilities. Out-of-the-box,
+ElasicSearch supports backing up to filesystems, and this is the only approach
```
Comment (duncan-ncc) [inline docs/backups.md]: Elasticsearch not ElasicSearch. Capitalisation and spelling

File: docs/backups.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+supported by LME. Other backup destinations are supported but these require
+separate plugins, and are not supported by LME.
+
+## Approach
+
+Backups are created using ElasticSearch snapshots. The initial snapshot will
```
Comment (duncan-ncc) [inline docs/backups.md]: Capitalisation, Elasticsearch is a trademark and should only have the E capitalised.

File: docs/backups.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+since the last snapshot was taken. It is therefore possible to take regular
+backups without a significant effect on the system's performance and without
+consuming large amounts of disk space.
+
+## Setting up a backup schedule
+
+### Create a filesystem repository
+
+The LME installation creates a bind mount in Docker that maps to the
+`/opt/lme/backups` directory on the host system. *TODO* CONFIRM THIS LOCATION
```
Comment (duncan-ncc) [inline docs/backups.md]: TODO still present

File: docs/backups.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Once the external drive has been mounted on the host, you need to create a
+repostiory for Elastic to use, which can be done through the Kibana interface.
+
+First navigate to the "Snapshot and Restore" page under the management tab:
+![Snaphost and Restore](backup_pics/snapshot_and_restore.png)
+
+Then create a repository by clicking the "Register a repository" button and
+filling in the following screens:
+
+![Repostiory one](backup_pics/repository_1.png)
```
Comment (duncan-ncc) [inline docs/backups.md]: Typo

File: docs/backups.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+In the above picture, the repository has been named "LME-backups" but you can
+select any other name as appropriate. The "Shared file system" repository type
+should be selected.
+
+On the next screen, the file system location should be set to
+`/opt/lme/backups`. The other fields can be left with the default values.
+
+![Repostiory two](backup_pics/repository_2.png)
```
Comment (duncan-ncc) [inline docs/backups.md]: Typo

File: docs/backups.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+The repository will be created and will show in the list on the management
+screen:
+
+![Repostiory three](backup_pics/repository_3.png)
```
Comment (duncan-ncc) [inline docs/backups.md]: Typo

File: docs/backups.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+You then need to create a policy for the backups. Select the "policies" tab and
+then click the "Create a policy" button:
+
+![Policy One](backup_pics/policy_1.png)
+
+On the next screen, pick a name for your new policy ("lme-snapshots" in this
+example). For the snapshot name the value `<lme-daily-{now/d}>` will create
+files with the prefix `lme-daily` and with the current date as a suffix. Make
+sure your new repository is selected, and then configure a schedule in line with
+your backup policy. Elastic Search uses incremental snapshots for its backup,
```
Comment (duncan-ncc) [inline docs/backups.md]: Elasticsearch

**Review by duncan-ncc — CHANGES_REQUESTED** [review]

File: docs/backups.md (hunk 3 above)
Comment (lllama) [reply docs/backups.md]: Assuming we're happy with this location

**Review by duncan-ncc — APPROVED** [review]
Thank you for resolving the comments.

File: docs/backups.md (hunk 7 above)
Comment (a-d-a-m-b) [reply docs/backups.md]: Line 64's Elastic Search needs to be amended

**Review by a-d-a-m-b — CHANGES_REQUESTED** [review]
docs/backups.md. Line 64. Elastic Search needs be changed to Elasticsearch

File: docs/backups.md (hunk 7 above)
Comment (lllama) [reply docs/backups.md]: Fixed

**Review by a-d-a-m-b — APPROVED** [review]
Thanks for the changes!

**Pull request merged**